plugins/mcreator-core/procedures/$projectileactions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "color": 270

```

Comment:
I think you should put a blue color, as projectiles are an entity type. In addition, this color should be used for the DyeColor variable, so it can be confusing for users.

Reply:
The main issue I have with blue is that there are already a lot of types with a blue color (text, entity, player, number, logic, custom code), so it might be difficult to see the exact hue

Reply:
Text blocks are green and custom code blocks are purple. I agree several block types are blue, but they are enough different to differentiate each block type.

Reply:
I changed the color to 290, so it shouldn't conflict with the dye procedures anymore. I kept a purple color because they're supposed to be used only with the shoot procedures, and because otherwise it'd still be too similar to the existing colors (the highest color difference it could have is 10, which imo is too little)